docs/tutorial.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 3
+---
+
+
+# Vitessce Link Tutorial with Sample Dataset
+
+To load a 3D dataset first go to [vitessce.link/study](file:///Users/ericmoerth/Library/CloudStorage/GoogleDrive-eric.moerth@gmx.at/My%20Drive/HARVARD%20MEDICAL%20SCHOOL/Reviews/BioMedVis2024/vitessce.link/study) on your Desktop   
```

Comment:
This is a local file, and the URL appears to be outdated (i.e. we don't have a `/study` url). I think `/beta` is the intended link?
```suggestion
To load a 3D dataset first go to [vitessce.link/beta](/beta) on your Desktop   
```